ts/components/screens/BaseScreenComponent.tsx

```diff
@@ -170,6 +176,20 @@ class BaseScreenComponent extends React.PureComponent<Props, State> {
     customHandledLink.map(link => handleItemOnPress(link)());
   };
 
+  private chMarkdown = (chTitle: string, chBody: string) => ({
```

Comment:
Is this change needed?

Reply:
Not anymore after this PR https://github.com/pagopa/io-app/pull/2363